client/app/startup/ManageAvailabilityApp.js

```diff
@@ -24,7 +24,6 @@ export default (props) => {
     manageAvailability: Immutable.Map({
       isOpen: window.location.hash.replace(/^#/, '') === EDIT_VIEW_OPEN_HASH,
       visibleMonth: moment()
-        .utc()
```

Comment:
If we shouldn't use `.utc()`, that should probably be removed from the two places it's used in `ManageAvailabilityReducer.js` as well?